### PR TITLE
Add SPDX Implementers Team Meeting - 22 Mar, 2023

### PR DESCRIPTION
**Pull request by rnjudge**

### CODE REVIEW

File: implementers/2023-03-22.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Notes
+### SPDX 3.0 Implementation planning
+* A lot of tool maintainers would like to deprecate 2.2 but it's an ISO standard which makes it hard
+* 3.0 will  be the next ISO standard
+
+### Open Table
+* sbom-composer tool was donated to opensbom-generator namespace: https://github.com/opensbom-generator/sbom-composer
+* The tool doesn't currently support CDX but plans for it to have support in the future
+* There is a java tool that converts SPDX to CDX: https://github.com/spdx/cdx2spdx
```
Comment (goneall) [inline implementers/2023-03-22.md:25]: Actually, this converts CDX to SPDX - the conversion from SPDX to CDX has been integrated into the C# CycloneDX CLI: https://github.com/CycloneDX/cyclonedx-cli - suggest replacing the above with "There is a C# tool that converts SPDX to CDX: https://github.com/CycloneDX/cyclonedx-cli

**Review by goneall — CHANGES_REQUESTED** [review]
Sorry about the confusion on the CDX/tools - one correction request

File: implementers/2023-03-22.md (hunk 1 above)
Comment (rnjudge) [reply implementers/2023-03-22.md]: Thanks @goneall - Can you take a look at the changes I pushed?

**Review by goneall — APPROVED** [review]
Thanks @rnjudge - looks good

**Pull request merged**